docs/source/getting_started/primitives.ipynb

```diff
@@ -179,14 +190,14 @@
    ]
   },
   {
-   "cell_type": "raw",
+   "cell_type": "markdown",
    "metadata": {
-    "raw_mimetype": "text/restructuredtext"
+    "raw_mimetype": "text/markdown"
    },
    "source": [
-    "Above, we created a new transform primitive that can be used with Deep Feature Synthesis by deriving a new primitive class using :class:`TransformPrimitive` as a base and overriding :meth:`get_function` to return a function that calculates the feature. Additionally, we set the input data types that the primitive applies to and the return data type.\n",
+    "Above, we created a new transform primitive that can be used with Deep Feature Synthesis by deriving a new primitive class using `TransformPrimitive` as a base and overriding `get_function` to return a function that calculates the feature. Additionally, we set the input data types that the primitive applies to and the return data type.\n",
```

Comment:
Maybe worth a short mention that input an return types are defined by using a Woodwork ColumnSchema to specify the logical type and/or semantic tags that are valid for input/output. Don't need a lot of detail, maybe just a sentence and then a link to the Woodwork types and tags documentation: https://woodwork.alteryx.com/en/stable/guides/understanding_types_and_tags.html